### PR TITLE
Update Stanford footer links

### PR DESCRIPTION
**Pull request by caesar2164**
example screenshot:

screen shot 2019-01-09 at 13 42 30

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 Reminder that before shots are usually helpful here for comparison :)

**Comment by stvstnfrd** [comment]
Also, I think this kind of change would benefit from a slightly more detailed description.
Sure you're "Updat[ing the] Stanford footer links" but _what_ actually changed?
Did you add a link? Remove one? Change the styling? etc...

**Comment by caesar2164** [comment]
My bad, I thought lagunita.stanford.edu would have worked as a before, here is the before:
screen shot 2019-01-09 at 16 06 17

File: lagunita/lms/templates/su-footer.html [hunk 1]
```diff
@@ -23,10 +23,12 @@
           <li class="emergency alt"><a href="https://emergency.stanford.edu">Emergency Info</a></li>
         </ul>
         <ul id="policy-links">
-          <li class="terms alt"><a href="https://www.stanford.edu/site/terms/">Terms of Use</a></li>
```
Comment (stvstnfrd) [inline lagunita/lms/templates/su-footer.html:26]: Where else is this `.terms` class being referenced?
Is it it in CSS?

Comment (caesar2164) [reply lagunita/lms/templates/su-footer.html:26]: It isn't, that's why I'm removing it. The `alt` class is being used in the CSS to break each link in mobile though.

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**